core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.scss

```diff
@@ -0,0 +1,53 @@
+@use "variables" as *;
+
+form {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    justify-content: center;
+}
+
+.ai-image-prompt__input {
+    display: flex;
+    flex-direction: column;
+
+    > div {
+        display: flex;
+        align-items: center;
+    }
+
+    > span {
+        display: flex;
+        margin: $spacing-0 0 $spacing-1 $spacing-5;
+        font-size: $font-size-sm;
+        color: $color-palette-gray-700;
+    }
+}
+
+.ai-image-prompt__text {
```

Comment:
Since we are using BEM, do we need this level of nested class?

Reply:
No we don't.